store/kv/kv_store.go

```diff
@@ -86,9 +93,25 @@ func (tx *KeyValueTx) ReadRange(ctx context.Context, table []byte, lkey Key, rke
 	if err != nil {
 		return nil, err
 	}
+
 	return NewKeyValueIterator(ctx, iter), nil
 }
 
+func (tx *KeyValueTx) GetMetadata(ctx context.Context, table []byte, key Key) (*internal.TableData, error) {
+	b, err := tx.ftx.Get(ctx, getFDBKey(table, key), true).Get()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(b) == 0 {
+		return nil, ErrNotFound
+	}
+
+	log.Debug().Int("length", len(b)).Msg("getMetadata")
+
+	return internal.Decode(b)
```

Comment:
so is it decoding only attributes?

Reply:
It decodes everything. But it's O(1) so no need in special decoding.

Reply:
great